proto/storage/compliance_operator_v2.proto

```diff
@@ -238,3 +238,21 @@ message ComplianceOperatorSuiteV2 {
 
   ComplianceOperatorStatus status = 4;
 }
+
+// Next Tag: 6
+message ComplianceOperatorBenchmark {
+  string id = 1; // @gotags: sql:"pk,id,type(uuid)"
+  string name = 2; // @gotags: search:"Compliance Benchmark Name,hidden"
+  string version = 3;
+  string description = 4;
+  string profile_label = 5;
+}
+
+// TODO: Combined primary key for benchmark_id and rule_id
+// Next Tag: 4
+message ComplianceOperatorControl {
+  string id = 1; // @gotags: sql:"pk,id,type(uuid)"
+  string benchmark_id = 2;
+  string rule_id = 3;
```

Comment:
One control can have multiple rules. 
A Rule can be implemented by multiple controls.

```
message ComplianceOperatorControl {
  string id = 1;
  string benchmark_id = 2;
  string rule_control_id = 3;
}

message ControlRule {
  string rule_id = 1;
  string control_id = 2;
}
```